src/openrct2/Game.cpp

```diff
@@ -588,6 +589,20 @@ void game_load_init()
     gGameSpeed = 1;
 }
 
+void game_load_scripts()
+{
+#ifdef __ENABLE_SCRIPTING__
+    GetContext()->GetScriptEngine().LoadPlugins();
+#endif
+}
+
+void game_finish()
```

Comment:
If this is just intended for unloading plugins for now, it should have a name that better reflects this, maybe `game_unload_scripts()`